package.json

```diff
@@ -9,7 +9,7 @@
     "npm": "8.5.5"
   },
   "devDependencies": {
-    "@tacc/core-styles": "git+https://git@github.com/TACC/Core-Styles.git",
+    "@tacc/core-styles": "git+https://git@github.com/TACC/Core-Styles.git#task/fp-1648-build-all-files-and-commit-dist",
```

Comment:
I assume this branch hash will be removed before merge

Reply:
Yes. Thanks. I need to think of a better way to do this for PRs (this is helpful for testing). I could use commits instead, like https://github.com/TACC/Core-CMS/pull/493#discussion_r880921494. Thoughts?

Reply:
Pinned to tag `v0.5.0` (66c6d14).

Long-term, I guess I need to:
- **either** version often (easier if I can independently test core-styles)
- **or** point to commits